assemblies/rosa-manage-objects-cli.adoc

```diff
@@ -0,0 +1,20 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-managing-objects-cli"]
+
+= Managing objects with `rosa`
```

Comment:
Per recent discussion, no formatting in titles

Reply:
Fixed.